magento/magento-functional-tests-migration
# Convert CaptchaEditCustomerTest to MFTF

### Description (*)

This PR with a test that checks captcha and lockout customer on the account edit page.
Steps:

Preconditions:
 1. Enable CAPTCHA for customer edit page.
 2. Create customer.

Test Flow:
1. Log in to Store Front.
2. Open customer account edit page.
3. Update email with incorrect password 3 or more times.
4.  Check captcha visibility after incorrect password submit the form 
5. Submit the form with incorrect captcha .

### Fixed Issues 

1. #248 Convert CaptchaEditCustomerTest to MFTF
2. Fixed changeEmail and changePassword incorrect element selectors in app/code/Magento/Customer/Test/Mftf/Section/StorefrontCustomerAccountInformationSection.xml
